plugins/module_utils/repo_setup/get_hash/constants.py

```diff
@@ -77,9 +67,9 @@
         "security",
         "swift",
         "tempest",
-        "tripleo",
+        "podified",
         "ui",
         "validation",
     ],
-    "os_versions": ["centos7", "centos8", "centos9", "rhel8", "rhel9"],
+    "os_versions": ["centos8", "centos9", "rhel8", "rhel9"],
```

Comment:
if you are just keeping podified and osp-18, antelope and master, do we need centos8 and rhel8?

Reply:
We will dropp CentOS/RHEL-8 related stuff in a seperate pr.